diff --git a/lib/env/main.js b/lib/env/main.js
index aabc07553af5ef2e1631d99039fc2e457ea780e0..4f8e39ca74b46f662f529240f433955ec518ac31 100644
--- a/lib/env/main.js
+++ b/lib/env/main.js
@@ -1,4 +1,5 @@
-import omit from 'omit.js';
+import omitModule from 'omit.js';
+const omit = omitModule.default || omitModule;
 import { removeFalsy } from '../utils/remove_falsy.js';
 import { getEnvelope } from './envelope.js';
 import { getGitEnv } from './git.js';
@@ -144,7 +145,7 @@ const getConfigFileEnv = function ({ config: { build: { environment = {} }, }, }
 };
 // Some environment variables cannot be overridden by configuration
 const cleanUserEnv = function (userEnv) {
-    return omit.default(userEnv, READONLY_ENV);
+    return omit(userEnv, READONLY_ENV);
 };
 const READONLY_ENV = [
     // Set in local builds
